specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.2-preview/keys.json

```diff
@@ -1870,6 +1870,24 @@
           "type": "string",
           "format": "base64url",
           "readOnly": true
+        },
+        "iv": {
+          "x-ms-client-name": "result",
```

Comment:
Can't have the same client name (same as below). I'd just elide the property here.

Reply:
fixed